torch/nn/functional.py

```diff
@@ -613,15 +625,35 @@ def alpha_dropout(input, p=0.5, training=False, inplace=False):
     return f(input, p, training)
 
 
-def dropout2d(input, p=0.5, training=False, inplace=False):
+def dropout2d(input, p=0.5, training=True, inplace=False):
+    r"""
+    Randomly zeroes whole channels (a channel a 2D slice of dimensions H, W) of the input tensor.
+
+    See :class:`~torch.nn.Dropout2d` for details.
+
+    Args:
+        p: probability of an element to be zeroed. Default: 0.5
+        training: apply dropout if is True. Defualt: True
+        inplace: If set to ``True``, will do this operation in-place. Default: ``False``
+    """
     if p < 0 or p > 1:
         raise ValueError("dropout probability has to be between 0 and 1, "
                          "but got {}".format(p))
     f = _VF.feature_dropout_ if inplace else _VF.feature_dropout
     return f(input, p, training)
 
 
-def dropout3d(input, p=0.5, training=False, inplace=False):
+def dropout3d(input, p=0.5, training=True, inplace=False):
+    r"""
+    Randomly zeroes whole channels (a channel is a 3D slice of dimensions D, H, W) of the input tensor.
```

Comment:
We shouldn't refer to D H W here, it isn't defined in this context and only makes sense in the nn doc.

Reply:
yeah. although the original doc indeed wasn't very clear on what is "channel", and whether each sample in a batch can have different channels zeroed out.

Reply:
I agree. I will remove `D H W` here. And maybe we should define "channel" at `conv` ops. By "different channels", do you mean "zeros a channel independently"?

Reply:
yep. 1st sample can have its 1st channel zeroed out, while the 2nd sample doesn't.